directories/src/redux/services/broker-node.js

```diff
@@ -2,38 +2,99 @@ import axios from "axios";
 import { API_HEADERS, API_ROOT_URL, API_REQUEST_ERROR } from "../../config/";
 
 const registerWebnode = address =>
-  axios({
-    method: "POST",
-    url: `${API_ROOT_URL}/api/v2/supply/webnodes`,
-    data: { address }
+  new Promise((resolve, reject) => {
+    axios({
+      method: "POST",
+      url: `${API_ROOT_URL}/api/v1/supply/webnodes`,
+      data: { address }
+    })
+    .then(data => {
+      resolve({ data });
+    })
+    .catch(error => {
+      console.log("POST registerWebnode" + error);
+      reject(API_REQUEST_ERROR);
+    });
   });
 
-const requestBrokerNodeAddressPoW = currentList =>
-  axios({
-    method: "POST",
-    url: `${API_ROOT_URL}/api/v2/demand/transactions/brokernodes`,
-    data: { currentList }
+const claimTreasure = seed =>
+  new Promise((resolve, reject) => {
+    axios({
+      method: "POST",
+      url: `${API_ROOT_URL}/api/v1/supply/claimtreasure`,
+      data: { seed }
```

Comment:
I am not sure, but I think we may have to give it more than just the seed.  I think we'd have to give it the genesis hash, file size (or # of chunks), and the sector we did the PoW for.  We won't necessarily be holding onto these seeds long-term so the broker will need some way to actually build the datamap and know what addresses to check to confirm the PoW was done.

Reply:
Ok, so we will add only filesize and genesishash to API calls? @EdmundMai do you aggree?